### PR TITLE
Add Aquarium addon documentation

### PR DESCRIPTION
**Pull request by ymoriaud**
Hi everyone 👋 

I'm adding specific documentation for a new incoming addon : [Aquarium](https://fatfi.sh/aquarium).

Aquarium addon logo

In this PR, you will find:

- Admin documentation regarding installation and configuration
- Developer documentation to setup dev environment and understand how the code is structured
- Project manager documentation explaining how to create, pair and sync projects
- (Small) artist documentation
- Images to illustrate the documentation (Optimize with ImageOptim to try to keep them as small as possible)

Thanks !

### CODE REVIEW

File: website/sidebars.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
+function sorted(a, b) {
```
Comment (ymoriaud) [inline website/sidebars.js:1]: I took the liberty to create this function to order the sidebar alphabetically. It's not changing the order specified in each items categories. Just the root level.

File: website/docs/addon_aquarium_manager.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+![Create on Aquarium](assets/aquarium/create-on-aquarium.png)
+
+Clicking on the button will :
+
+- Gather your Ayon hierarchy
+- Generate a compatible JSON file with the project, templates, entities and tasks
+- Upload that JSON file to Aquarium
+
+:::info
+This process is asynchronous. So if your Ayon project is really big, the import will take few minutes. Fow now, the only way to track the completion of the import, is to use Aquarium API, to list all the jobs of your bot.
```
Comment (MustafaJafar) [inline website/docs/addon_aquarium_manager.md:75]: There's a typo here.
```suggestion
This process is asynchronous. So if your Ayon project is really big, the import will take few minutes. For now, the only way to track the completion of the import, is to use Aquarium API, to list all the jobs of your bot.
```

**Review by MustafaJafar — APPROVED** [review]
This is so cool!
This is a great example of how to contribute to Ayon docs! 
Many Thanks.

File: website/sidebars.js (hunk 1 above)
Comment (MustafaJafar) [reply website/sidebars.js:1]: This is helpful!

File: website/docs/addon_aquarium_manager.md (hunk 2 above)
Comment (ymoriaud) [reply website/docs/addon_aquarium_manager.md:75]: Thanks ! :)

File: website/docs/addon_aquarium_developer.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  - Frontend
+- [Services](#services)
+  - Leecher
+  - Processor
+
+### Directory
+
+#### Client
+
+Client code is used in the [Ayon Launcher](ayon-launcher), the AYON pipeline desktop application.
```
Comment (MustafaJafar) [inline website/docs/addon_aquarium_developer.md:51]: Hello, 
The link `[Ayon Launcher](ayon-launcher)` is dead.
Was it supposed to point to the ayon-launcher repo?
```suggestion
Client code is used in the [Ayon Launcher](https://github.com/ynput/ayon-launcher), the AYON pipeline desktop application.
```

Comment (ymoriaud) [reply website/docs/addon_aquarium_developer.md]: Thanks ! I approved your suggestion. :)

**Review by mkolar — APPROVED** [review]

**Pull request merged**